src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/page.tsx

```diff
@@ -36,6 +37,13 @@ export default async function SettingsPage({ searchParams }: Props) {
   const session = await getServerSession();
   console.debug(searchParams);
 
+  if (session?.user?.subscriber) {
+    const unsubLink = await generateUnsubscribeLinkForSubscriber(
+      session.user.subscriber,
+    );
+    console.info({ unsubLink });
```

Comment:
This is only for testing right? I think we should only generate the link when the email is crafted